pkg/tests/smoke/testdata/Bob/test.gpt

```diff
@@ -1,10 +1,10 @@
 tools: bob
 
-Ask Bob "how are you doing" and repeat his reply exactly.
+Ask Bob "how are you doing" and repeat the response text exactly as given without saying anything else.
 
 ---
 name: bob
 description: I'm Bob, a friendly guy.
 args: question: The question to ask Bob.
 
-When asked how I am doing, respond with exactly "Thanks for asking "${QUESTION}", I'm doing great fellow friendly AI tool!"
+When asked how I am doing, respond with the following exactly: "Thanks for asking '${question}'! I'm doing great fellow friendly AI tool!" with ${question} replaced with the question text as given.
```

Comment:
yup, we did.

Fixed and pushed.

Reply:
curious if this makes any functional difference here since this isnt a code tool - `${QUESTION}` is being made to look like an environment variable here, but there isnt anything actually setting or reading env vars... its just the llm being a smarty pants.

Reply:
Yeah, this is more for convention sake than anything.